shared/js/ui/base/chrome-ui-wrapper.es6.js

```diff
@@ -28,34 +30,34 @@ let getBackgroundTabData = () => {
     })
 }
 
-let search = (url) => {
-    window.chrome.tabs.create({url: `https://duckduckgo.com/?q=${url}&bext=${window.localStorage['os']}cr`})
+const search = (url) => {
+    const os = parseUserAgentString().os
```

Comment:
instead of pulling OS from localStorage I'm calling the method that pulls that info from UA